fuzz/fuzz_targets/validate-valid-module.rs

```diff
@@ -1,11 +1,23 @@
 #![no_main]
 
+use arbitrary::Arbitrary;
 use libfuzzer_sys::fuzz_target;
-use wasm_smith::Module;
+use wasm_smith::ConfiguredModule;
+
+#[derive(Default, Debug, Arbitrary)]
+struct Config {
+    memories: u8,
+}
+
+impl wasm_smith::Config for Config {
+    fn max_memories(&self) -> u32 {
+        std::cmp::min(3, self.memories.into())
+    }
+}
 
 // Define a fuzz target that accepts arbitrary
 // `Module`s as input.
-fuzz_target!(|m: Module| {
+fuzz_target!(|m: ConfiguredModule<Config>| {
```

Comment:
Maybe just use `ConfiguredModule<SwarmConfig>`? I have a local commit that adds multi-memory support to `SwarfmConfig`, PR up in a minute.

Reply:
Oh nice I didn't realize that, updated!